client/gutenberg/extensions/giphy/settings.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { Path, SVG } from '@wordpress/components';
+
+export const settings = {
+	name: 'giphy',
+	icon: (
+		<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+			<Path fill="none" d="M0 0h24v24H0V0z" />
+			<Path d="M17.343 9.171h-4.114V5.057H7.057v13.886h10.286V9.17zm2.057-.514V21H5V3h10.286v2.057h2.057v2.057H19.4v1.543z" />
+		</SVG>
+	),
+	prefix: 'jetpack',
+	title: __( 'Giphy' ),
+	category: 'jetpack',
+	keywords: [ __( 'giphy' ) ],
+	description: __( 'Add a giphy graphic.' ),
+	attributes: {
+		align: {
+			type: 'string',
+			default: 'center',
+		},
+		caption: {
+			type: 'string',
+		},
+		giphyUrl: {
```

Comment:
I have no problem doing this or the requested padding percentage change, but they seem to be subtly contradictory. In the padding case, you're requesting a change that increases the length of the stored attribute so that it can be used as-is without any programmatic modification. In this case the suggestion seems to be the opposite—store the shorter, essential value and transform programmatically when the value is needed for rendering. Again, a very subtle point and I'm happy to make both changes as requested :-), I'm just trying to grasp the full nature of the recommendation.

Reply:
That's a good point and thanks for mentioning it here

🤔 💭 

After much careful consideration 🤕, let's store the padding as a complete percent string and forget about this change, storing the full giphy URL is good 👍